test/controllers/commit-controller.test.js

```diff
@@ -87,6 +87,16 @@ describe('CommitController', function() {
       assert.strictEqual(wrapper.find('CommitView').prop('message'), 'some message');
     });
 
+    it('repository.didUpdate is not called when commit message changes', function() {
```

Comment:
This is very minor, but it looks like all of the surrounding tests' descriptions are phrased so that they sound right after the word "it". I'd maybe tweak the description to say something like

`it("does not cause the repository to update")`.